utils/dbutils.py

```diff
@@ -0,0 +1,151 @@
+from gempython.utils.gemlogger import getGEMLogger, printYellow
+
+import pandas as pd
+
+dbNameDev = "INT2R" # development
+dbNamePro = "cms_omds_lb" # production
+
+connection = "oracle://CMS_GEM_APPUSER_R:GEM_Reader_2015@"
+
+knownViews = [
+        'GEM_SUPRCHMBR_VFAT_VIEW',
+        #'GEM_VFAT2_CHIP_CONF_V_RH',
+        'GEM_VFAT3_CHIP_CONF_V_RH',
+        'GEM_VFAT3_PROD_SUMMARY_V_RH'
+        ]
+
+def getGemDBView(view, vfatList=None, fromProd=True, debug=False):
```

Comment:
Standard comment about function/variable naming conventions... `Gem` -> `GEM` when appearing not as the first part of said function/variable (also elsewhere)